packages/nextra/src/index.js

```diff
@@ -1,17 +1,16 @@
 import { NextraPlugin, pageMapCache } from './plugin'
 import { MARKDOWN_EXTENSION_REGEX } from './constants'
-import { LoaderOptions, WithNextra } from './types'
 
-const DEFAULT_EXTENSIONS = ['js', 'jsx', 'ts', 'tsx'] as const
-const MARKDOWN_EXTENSIONS = ['md', 'mdx'] as const
```

Comment:
hm, previously this file worked well with typescript, just need to use `module.exports` and not `export default`

Reply:
Before:
<img width="565" alt="Screen Shot 2022-07-24 at 02 16 01" src="https://user-images.githubusercontent.com/15878786/180617941-9cea8b2c-e3a1-4312-bfe5-bf9409980f46.png">

After:
<img width="554" alt="Screen Shot 2022-07-24 at 02 19 04" src="https://user-images.githubusercontent.com/15878786/180617946-9d18092a-a59d-4662-ac6a-2bef4d610ff3.png">